site/docs/iceberg/trino.md

```diff
@@ -28,4 +28,13 @@ s3.aws-secret-key=<aws_secret_key>
        a. To create branches and tags, use the Nessie CLI available [here](https://projectnessie.org/nessie-latest/cli/).
        b. To switch references (branch or tag), update the catalog property `iceberg.nessie-catalog.ref`.
 
+Nessie can also be accessed via REST catalog path.
+Sample Nessie-REST configuration for Iceberg connector in `etc/catalog/iceberg.properties`
```

Comment:
nit: Maybe put `:` at the end? (also on the similar line above)

Reply:
done